package/yast2-printer.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Dec 12 14:34:53 UTC 2018 - Stefan Hundhammer <shundhammer@suse.com>
+
+- Security hardening (bsc#1118291)
+- 4.0.4
```

Comment:
wrong version. IT should be 4.1.0 as it differs from SLE15-GA. see https://yastgithubio.readthedocs.io/en/latest/versioning/

Reply:
Changed.